File: java/Q37.java
import java.util.Scanner;

public class Q37 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter number : ");
        int num = sc.nextInt();
        int rev = 0;

        while(num > 0){
            int rem = num % 10;
            rev = rev * 10 + rem;
            num /= 10;
        }
        if(num == rev){
            System.out.println("This is palindromic");
        }
        else {
            System.out.println("This is not palindromic");
        }
    }
}

File: Javascript/Q37.js
const prompt = require("prompt-sync")();

let num = +prompt("Enter number : ");
let rev = 0;
let actual = num;

while (num > 0) {
  let rem = num % 10;
  rev = Math.floor(rev * 10) + rem;
  num = Math.floor(num / 10);
}

if (actual == rev) {
  console.log("This is palindromic number");
} else {
  console.log("This is not palindromic number");
}
